### DISCUSSION TITLE
How can low-residency grad students renew their student developer pack without any documentation that states "distance learner"?

### DISCUSSION THREAD

**Discussion by Linsleysk** [community/community#49610] [GitHub Education]
I am trying to renew my student developer pack, but the application is more difficult to pass this time. I am a low-residency MFA student and am only on the college campus twice per year, so my application was denied since my current laptop location is not in the same state as the campus. The application requires I provide college-issued documentation stating that I am a distance learner, but this program only uses the term "low-residency", not distance learning. GITHUB no longer offers support for these issues & exceptions. Anyone have suggestions? Thank you!

**Comment by samus-aran** [comment]
Hi @Linsleysk 👋 

The application process does takes in other deciding factors than just the location. We also want those applying to acquire (if possible) and ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Let us know how you get on 👍
